actions.yaml

```diff
@@ -0,0 +1,5 @@
+# Copyright 2024 Canonical Ltd.
+# See LICENSE file for licensing details.
+
```

Comment:
I have removed the `get-xxx-credentials` action together with `sync-xxx-credentials` action here, because now that we set secret directly through config, I don't see any reason we should provide a dedicated action to see the parameters. One can simply do `juju config object-storage-integrator` to see the parameters.